src/ORM/ResetDatabase/OrmDatabaseResetterTrait.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Zenstruck\Foundry\ORM\ResetDatabase;
+
+use Doctrine\Bundle\DoctrineBundle\Registry;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Platforms\PostgreSQLPlatform;
+use Doctrine\DBAL\Platforms\SQLitePlatform;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+/**
+ * @internal
+ * @author Nicolas PHILIPPE <nikophil@gmail.com>
+ */
+trait OrmDatabaseResetterTrait
+{
+    protected function dropAndResetDatabase(Application $application): void
+    {
+        foreach ($this->connections() as $connectionName) {
```

Comment:
a real robust implementation would be to have a service per connection, but this sounds a bit overkill

Reply:
We'll wait for this to be an issue.

Reply:
yes of course